pandas/tests/indexes/timedeltas/test_arithmetic.py

```diff
@@ -453,10 +453,16 @@ def test_timedelta_ops_with_missing_values(self):
         # setup
         s1 = pd.to_timedelta(Series(['00:00:01']))
         s2 = pd.to_timedelta(Series(['00:00:02']))
-        sn = pd.to_timedelta(Series([pd.NaT]))
+        with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
```

Comment:
Can we get this stacklevel correct? Or are we calling this from two places with different stacklevels?

Reply:
Possibly not for both `to_timedelta` and `DatetimeIndex`, but so let's choose and eg take the correct stacklevel for the case of an array to `to_timedelta`

Reply:
I lost patience pretty quickly while trying.  It wouldn't surprise me that much if it turned out to be really simple ex-post.

Reply:
Can you do this as a follow-up as well?